web/src/views/SubmissionPortal/store/index.ts

```diff
@@ -110,6 +111,7 @@ const addressFormDefault = {
 const contextFormDefault = {
   dataGenerated: undefined as undefined | boolean,
   awardDois: [] as string[] | null,
+  ship: undefined as undefined | boolean,
```

Comment:
I think this also needs to be added to the pydantic class in `schemas_submission.py`. Otherwise users will have to select an option for this each time they open the submission.